benchmarks.yaml

```diff
@@ -200,4 +183,28 @@
   server: HonoJS
   run:
     cmd: bun
-    args: ["index.js"]
\ No newline at end of file
+    args: ["index.js"]
+- id: graphql-ruby
+  name: graphql-ruby
+  url: https://github.com/rmosolgo/graphql-ruby
+  lang: Ruby
+  server: Puma
+  run:
+    cmd: puma
+    args:
+      [
+        "-w",
+        <%= System.cpu_count.to_s %>,
+        "-s",
+        "-t",
+        "2",
+        "-b",
+        "tcp://127.0.0.1:8000",
+      ]
```

Comment:
Why re-order the servers?

Reply:
Check this one [Action](https://github.com/graphql-crystal/benchmarks/actions/runs/6630650384/job/18012480495#step:9:481)

Also quite some more PR's have been failed do you this
```
/home/runner/work/benchmarks/benchmarks/graphql-ruby: command 'bundle' failed with exit code 1
```

So I reordered it

Reply:
But I'm not quite sure, that reordering can fix that problem all time.